generate/test_data/test_data_msgp.go.out

```diff
@@ -0,0 +1,755 @@
+package test_data
+
+// Code generated by github.com/tinylib/msgp DO NOT EDIT.
+
+import (
+	"github.com/tinylib/msgp/msgp"
```

Comment:
No, because that would create an unnecessary dependency on a fork that we hope will be temporary.  Using our version would be appropriate if we decide to make the fork permanent, but otherwise it's better not to -- we can override the source of the upstream package to point to our fork in projects that use the generated MessagePack code.